rollup.config.js

```diff
@@ -1,19 +1,30 @@
-import path from 'path'
 import pkg from './package.json'
 
 export default [
   {
-    input: {
-      index: 'src/index.js',
-      matchers: 'src/matchers.js',
-    },
+    input: 'src/index.js',
```

Comment:
Maybe it's better to provide an array for an input? So rollup can share code between files, otherwise matchers are duplicated.

Reply:
I'm not terribly familiar with rollup.  How can I specify an array for input, when each one should have a specific different output?

Reply:
There is an example in Vitest repo: https://github.com/vitest-dev/vitest/blob/main/packages/vitest/rollup.config.js

You will need to entire, something like this:

```ts
export default [
{
  input: entries,
  output: {
    dir: 'dist',
    entryFileNames: '[name].mjs',
   format: 'esm'
  }
},
{
  input: entries,
  output: {
    dir: 'dist',
    entryFileNames: '[name].js',
   format: 'cjs'
  }
}
]
```